.github/workflows/conformance-clustermesh.yaml

```diff
@@ -302,6 +302,7 @@ jobs:
             --helm-set=hubble.tls.auto.certManagerIssuerRef.kind=Issuer \
             --helm-set=hubble.tls.auto.certManagerIssuerRef.name=cilium \
             --helm-set=clustermesh.useAPIServer=${{ matrix.mode != 'external' }} \
+            --helm-set=clustermesh.config.enabled=${{ matrix.mode != 'external' }} \
```

Comment:
Hmm, this should simply be `clustermesh.config.enabled=true`, because we are setting it to true anyways in external mode below (L552).